README.md

```diff
@@ -42,11 +54,56 @@ See our [example](./examples) notebooks for how to develop with `gordo` locally.
 
 ---
 
-## Install: 
+## Install 
 `pip install --upgrade gordo`  
 
 Bleeding edge:  
 `pip install git+https://github.com/equinor/gordo.git`
 
-## Uninstall:
+## Uninstall
 `pip uninstall gordo`
+
+## Developer manual
+This section will explain how to start development of Gordo.
+
+### How to prepare working environment
+- install requirements
+```shell script
+# create and activate virtualenv. Note: you should use python3.7 (project's tensorflow version is not compatible with python3.8)
+# then:
+pip install --upgrade pip
+pip install --upgrade pip-tools
+pip install -r requirements/full_requirements.txt
+pip install -r requirements/test_requirements.txt
+```
+
+### How to run tests locally
+
+#### Tests system requirements
+To run tests it's required for your system to has (note: commands might differ from your OS):
+- running docker process;
+- available 5432 port for postgres container:
+    - check if port is available: `sudo lsof -i tcp:5432`
+    - (if needed) check `postgres` process details with: `ps auxwww | grep postgres` 
+    - (if needed) stop postgres on 5432 port (depends on how you installed postgres and its version):
+    ```shell script
+    sudo -u postgres /Library/PostgreSQL/12/bin/pg_ctl -D /Library/PostgreSQL/12/data stop -s -m fast
```

Comment:
`/Library/PostgreSQL/` that is specific to MacOS. And I thought that PostgreSQL in unit tests runs on docker containers

Reply:
this is removed accordingly to another comment above.
But anyway it seems that port is [exposed from the container](https://github.com/equinor/gordo/blob/master/tests/conftest.py#L268). So maybe it's a good idea not to face such issues is to change exposed and used port in the tests.